Week3/homework/3-step3.js

```diff
@@ -3,23 +3,53 @@
 // use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
   // add your code here
-  return str;
+  var strOut = "";
+  for (var i = 0; i < num; i++) {
+    strOut = strOut + str;
+  }
+  return strOut;
 }
 
 console.log('for', repeatStringNumTimesWithFor('abc', 3));
 
 // use a 'while' loop
 function repeatStringNumTimesWithWhile(str, num) {
   // add your code here
-  return str;
+  var str = 0;
+
+  while (str < 3) {
+    str++;
+  }
+
+
+  // expected output: 3
+
 }
 
 console.log('while', repeatStringNumTimesWithWhile('abc', 3));
```

Comment:
When run, it gives undefined :)

Reply:
Generally, I notice you didn't test your code. This is something that might have a really bad impact on the team, when you will be working in a real team environment. Please make sure you test your code always before handing it. In most of the cases, no matter how experienced you are, the code will not run as expected. It is our responsibility to ALWAYS test the code, no matter how confident we are.